index.js

```diff
@@ -13,7 +13,7 @@ module.exports = function toCommonLogFormat (request, options) {
       userid = request.id,
       time = '[' + moment().strftime('%d/%b/%Y:%H:%M:%S %z') + ']',
       requestLine = '"' + [method, path, httpProtocol].join(' ') + '"',
-      statusCode = request.response.statusCode,
+      statusCode = request.response.statusCode || request.response.output.statusCode,
```

Comment:
This was the shortest implementation, but it could also explicitly check for a Boom error:

```js
statusCode = request.response.isBoom ? request.response.output.statusCode : request.response.statusCode,
```